<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/yagf.png</normaloff>:/yagf.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QFrame" name="frame_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>1</width>
               <height>32</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <widget class="QToolButton" name="selectButton">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>31</width>
                <height>31</height>
               </rect>
              </property>
              <property name="toolTip">
               <string>Select block</string>
              </property>
              <property name="locale">
               <locale language="English" country="UnitedStates"/>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/images/select.png</normaloff>:/images/select.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
             <widget class="QToolButton" name="enlargeButton">
              <property name="geometry">
               <rect>
                <x>40</x>
                <y>0</y>
                <width>31</width>
                <height>31</height>
               </rect>
              </property>
              <property name="toolTip">
               <string>Larger view</string>
              </property>
              <property name="locale">
               <locale language="English" country="UnitedStates"/>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/images/larger.png</normaloff>:/images/larger.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </widget>
             <widget class="QToolButton" name="decreaseButton">
              <property name="geometry">
               <rect>
                <x>70</x>
                <y>0</y>
                <width>31</width>
                <height>31</height>
               </rect>
              </property>
              <property name="toolTip">
               <string>Smaller view</string>
              </property>
              <property name="locale">
               <locale language="English" country="UnitedStates"/>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/images/smaller.png</normaloff>:/images/smaller.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </widget>
             <widget class="QToolButton" name="rotateCCWButton">
              <property name="geometry">
               <rect>
                <x>110</x>
                <y>0</y>
                <width>31</width>
                <height>31</height>
               </rect>
              </property>
              <property name="toolTip">
               <string>Rotate 90 CCW</string>
              </property>
              <property name="locale">
               <locale language="English" country="UnitedStates"/>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/images/rccw.png</normaloff>:/images/rccw.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </widget>
             <widget class="QToolButton" name="rotate180Button">
              <property name="geometry">
               <rect>
                <x>140</x>
                <y>0</y>
                <width>31</width>
                <height>31</height>
               </rect>
              </property>
              <property name="toolTip">
               <string>Rotate 180</string>
              </property>
              <property name="locale">
               <locale language="English" country="UnitedStates"/>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/images/revert.png</normaloff>:/images/revert.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </widget>
             <widget class="QToolButton" name="rotateCWButton">
              <property name="geometry">
               <rect>
                <x>170</x>
                <y>0</y>
                <width>31</width>
                <height>31</height>
               </rect>
              </property>
              <property name="toolTip">
               <string>Rotate 90 CW</string>
              </property>
              <property name="locale">
               <locale language="English" country="UnitedStates"/>
              </property>
              <property name="text">
               <string>Rotate 90 CW</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/images/rcw.png</normaloff>:/images/rcw.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </widget>
             <widget class="QToolButton" name="singleColumnButton">
              <property name="geometry">
               <rect>
                <x>210</x>
                <y>0</y>
                <width>31</width>
                <height>31</height>
               </rect>
              </property>
              <property name="toolTip">
               <string>Read as a single column</string>
              </property>
              <property name="locale">
               <locale language="English" country="UnitedStates"/>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/images/singlecolumn.png</normaloff>:/images/singlecolumn.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QScrollArea" name="scrollArea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>145</width>
                <height>486</height>
               </rect>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QTextEdit" name="textEdit">
          <property name="font">
           <font>
            <family>Times New Roman</family>
           </font>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionScan"/>
    <addaction name="actionRecognize"/>
    <addaction name="actionRecognize_All_Pages"/>
    <addaction name="action_Save"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionOnlineHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionScan"/>
   <addaction name="actionPreviousPage"/>
   <addaction name="actionNextPage"/>
   <addaction name="separator"/>
   <addaction name="actionRecognize"/>
   <addaction name="actionRecognize_All_Pages"/>
   <addaction name="separator"/>
   <addaction name="action_Save"/>
   <addaction name="actionCopyToClipboard"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionScan">
   <property name="icon">
    <iconset>
     <normaloff>:/images/scanner.png</normaloff>:/images/scanner.png</iconset>
   </property>
   <property name="text">
    <string>S&amp;can...</string>
   </property>
   <property name="statusTip">
    <string>Scanning images using XSane...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset>
     <normaloff>:/images/document_save_as.png</normaloff>:/images/document_save_as.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save text...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>:/images/document_open.png</normaloff>:/images/document_open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open Image...</string>
   </property>
   <property name="toolTip">
    <string>Open Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionRecognize">
   <property name="icon">
    <iconset>
     <normaloff>:/images/filefind.png</normaloff>:/images/filefind.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Recognize</string>
   </property>
   <property name="statusTip">
    <string>Recognizing text...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionChoose_Language">
   <property name="text">
    <string>Choose &amp;Language</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionPreviousPage">
   <property name="icon">
    <iconset>
     <normaloff>:/images/back.png</normaloff>:/images/back.png</iconset>
   </property>
   <property name="text">
    <string>Previous page</string>
   </property>
   <property name="toolTip">
    <string>Move to previous image</string>
   </property>
  </action>
  <action name="actionNextPage">
   <property name="icon">
    <iconset>
     <normaloff>:/images/forward.png</normaloff>:/images/forward.png</iconset>
   </property>
   <property name="text">
    <string>Next page</string>
   </property>
   <property name="toolTip">
    <string>Move to next image</string>
   </property>
  </action>
  <action name="actionOnine_help">
   <property name="text">
    <string>Online Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionOnlineHelp">
   <property name="text">
    <string>Online Help</string>
   </property>
  </action>
  <action name="actionCopyToClipboard">
   <property name="icon">
    <iconset>
     <normaloff>:/images/editcopy.png</normaloff>:/images/editcopy.png</iconset>
   </property>
   <property name="text">
    <string>Copy To Clipboard</string>
   </property>
   <property name="toolTip">
    <string>Copy recognized text to clipboard</string>
   </property>
  </action>
  <action name="actionRecognize_All_Pages">
   <property name="icon">
    <iconset>
     <normaloff>:/batch.png</normaloff>:/batch.png</iconset>
   </property>
   <property name="text">
    <string>Recognize &amp;All Pages</string>
   </property>
   <property name="toolTip">
    <string>Recognize All Pages</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
